.github/workflows/test-and-publish-nightly.yml

```diff
@@ -54,13 +56,17 @@ jobs:
         id: build
         run: |
           set -x
-          case "${{ matrix.runson.name }}" in
+          HASH=${GITHUB_SHA:0:7}
+          case "${{ matrix.runson.display }}" in
             macos*)
               brew install gcovr
-              make COMMIT_TAG=${GITHUB_SHA:0:7} FAT_BINARY=true release coverage -j
+              make COMMIT_TAG=$HASH FAT_BINARY=true release coverage -j
+            ;;
+            alpine*)
```

Comment:
If we no longer build on alpine, this case is not needed?

Reply:
Yes, and same for setup-java a bit above, thanks: 2cd2700f5a847e2f38e33b858afa03e1c5bb1fc8